database/migrations/2016_07_25_052444_AlterTableComponentGroupsAddVisibleColumn.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AlterTableComponentGroupsAddVisibleColumn extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('component_groups', function (Blueprint $table) {
+            $table->integer('visible')->after('order')->unsigned()->default(0);
+
+            $table->index('visible');
+            $table->engine = 'InnoDB';
```

Comment:
Wait. I'm an idiot, I just noticed this is an alteration... We don't need that anyway `:facepalm:` - apologies!